client/src/pages/Register.tsx

```diff
@@ -1,23 +1,53 @@
 import { useState } from "react";
 import { authStore } from "../stores/AuthStore";
+import Button from "../components/UIElements/Button";
+import Input from "../components/UIElements/Input";
+import { Link, useNavigate } from "react-router-dom";
+import Card from "../components/UIElements/Card";
 
 const Register = () => {
-    const [name, setName] = useState("");
-    const [email, setEmail] = useState("");
-    const [password, setPassword] = useState("");
-    
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
```

Comment:
Same as the login page. These states causing alot of rendering.
We should talk about it live just for sake of it.

Reply:
let's talk!